temp#1.py

```diff
@@ -0,0 +1,131 @@
+# Calista Adler and Brandon Zhao
+
+import pandas as pd
+import csv 
+
+# opening and reading data from records.csv file 
+data = pd.read_csv(r'/Users/calistaadler/Downloads/records.csv')
+
+# writing into new csv file 
+f = open('/Users/calistaadler/Downloads/tempout.csv', 'w')
+writer = csv.writer(f)
+
+# creates column titles 
+header = ['Timestamp', 'ModA Temp', 'ModB Temp', 'ModC Temp', 'Gate Driver Board Temp']
+writer.writerow(header)
+
+# used to parse through each row 
+i = 0
+n = 1
```

Comment:
`i` and `n` should become unnecessary once you make the other changes I suggested